app/server/views/basic-app.hbs

```diff
@@ -3,11 +3,16 @@
     <head>
         <meta charset="utf-8">
         <title>{{title}}</title>
-        <link rel="stylesheet" href="/css/basicApp.css">
+        <link rel="stylesheet" href="/css/basicApp.css"
     </head>
     <body>
         <div id="app">
-            <basic-app :app-name="'{{appName}}'" :title="'{{title}}'"></basic-app>
+            <app-header
+                :app-title="'{{appTitle}}'"
+                :course-title="'{{courseTitle}}'"
+                :wodin-version="'{{wodinVersion}}'">
+             </app-header>
+            <basic-app :app-name="'{{appName}}'"></basic-app>
         </div>
         <script src="/js/basicApp.umd.js"></script>
     </body>
```

Comment:
I've deliberately not done that, see comment in description. Since there's nothing dynamic about the header, and course name and wodin version aren't required by anything else in the app, it seemed simpler to do it this way. Particularly since they are at a higher level than app config, so wouldn't come back in the app config endpoint (so would need another endpoint, or to mangle the existing one). 

Also, once the next ticket I'm currently working on is done, that will refactor the top level components in the front end a bit to have a common container component, so that will enable the app header to more easily sit inside basic-app and not need declaring separately, so at that point we'll be able to just declare basic-app here, with the props for wodin version etc.